NEWS.md

```diff
@@ -1,3 +1,68 @@
+* AppArmor prompting: include prompt prefix in AppArmor rules if prompting is supported and enabled
+* AppArmor prompting: add common types, constraints, and mappings from AppArmor permissions to abstract permissions
+* AppArmor prompting: add path pattern parsing and matching
+* Registry views (experimental): rename from aspects to registries
+* Registry views (experimental): support reading registry views using snapctl
+* Registry views (experimental): restrict view paths from using a number as first character and view names to storage path style patterns
+* Snap components: support installing snaps and components from files at the same time (no REST API/CLI)
+* Snap components: support downloading components related assertions from the store
+* Snap components: support installing components from the store (no REST API/CLI)
+* Snap components: support removing components (REST API, no CLI)
+* Snap components: started support for component hooks
+* Kernel modules as components: support kernel modules as components
+* Refresh app awareness (experimental): add data field for /v2/changes REST API to allow associating each task with affected snaps
+* Refresh app awareness (experimental): use the app name from .desktop file in notifications
+* Allow re-exec when SNAP_REEXEC is set for unlisted distros to simplify testing
+* Generate GNU build ID for Go binaries
+* Add missing etelpmoc.sh for shell completion
+* Do not attempt to run snapd on classic when re-exec is disabled
+* Packaging/build maintenance for Debian sid, Fedora, Arch, openSuse
+* Add snap debug api command to enable running raw queries
+* Enable snap-confine snap mount directory detection
+* Replace global seccomp filter with deny rules in standard seccomp template
+* Remove support for Ubuntu Core Launcher (superseded by snap-confine)
+* Support creating pending serial bound users after serial assertion becomes available
+* Support disabling cloud-init using kernel command-line
+* In hybrid systems, apps can refresh without waiting for restarts required by essential snaps
+* Ship snap-debug-info.sh script used for system diagnostics
+* Add refresh-observe access to /v2/snaps/{name} REST API
+* Improve error messages when attempting to run non-existent snap
+* Switch to -u UID:GID for strace-static
+* Support enabling snapd logging with snap set system debug.snapd.{log,log-level}
+* Fix potential unexpected auto-refresh of snap on managed schedule
+* Fix potential segfault by guarding against kernel command-line changes on classic system
+* Fix proxy entries in /etc/environment with missing newline that caused later manual entries to not be usable
+* Fix offline remodelling by ignoring prerequisites that will otherwise be downloaded from store
+* Fix devmode seccomp deny regression that caused spamming the log instead of actual denies
+* Fix snap state lock leak during refresh unlink
```

Comment:
This fix was for the snap lock leaking not the snap state lock, https://github.com/snapcore/snapd/pull/14044
```suggestion
* Fix snap lock leak during refresh
```

Reply:
Thanks, will change on next iteration.

Reply:
fixed